### ISSUE TITLE
List Animals in animal-list page

### ISSUE THREAD

**Issue by saulyz** [pets-oss/pets-front#27]
Task depends on #22 
Simply list all the animals and link each item to its own page.
Listing can be simple, showing only: id, name, mod_time (unless the UI is already made with more concrete design).
DataTable, Pagination and other presentation features are the object of another task.

**Comment by saulyz** [comment]
Issue is closed as things were done via #22 #32
